### ISSUE TITLE
Add support for Retina/HiDPI displays

### ISSUE THREAD

**Issue by jabooth** [menpo/landmarker.io#94]
This could maybe be a case of simply [flicking a setting on `THREE.WebGLRenderer`](http://stackoverflow.com/questions/27929979/three-js-webgl-renderer-for-high-dpi-displays-bug):

```
renderer = new THREE.WebGLRenderer({ devicePixelRatio: window.devicePixelRatio || 1 });
```

it also might be worth updating THREE.js at the same time to take advantage of any recent improvements.

**Comment by jabooth** [comment]
Based on that stack overflow r70+ has better handling of HiDPI, so we could bump to r71 which is a nice fresh release:
https://github.com/mrdoob/three.js/releases

**Comment by jabooth** [comment]
closed by fbea0f8077ad1d5621e314f734853f8744156267.
